src/Paket.Core/PaketConfigFiles/ProjectFile.fs

```diff
@@ -729,7 +731,7 @@ module ProjectFile =
             propertyNames,propertyGroup        
 
         let allTargets =
-            model.GetReferenceFolders()
+            model.CompileLibFolders
```

Comment:
Originally I wanted to leave the warnings here to review the warnings later and is we could do things differently